tools/syz-declextract/declextract.go

```diff
@@ -46,7 +47,7 @@ func main() {
 	loadProbeInfo := func() (*ifaceprobe.Info, error) {
 		return probe(cfg, *flagConfig)
 	}
-	if _, err := run(filepath.FromSlash("sys/linux/auto.txt"), loadProbeInfo, &clangtool.Config{
+	if _, err := run(filepath.FromSlash("sys/linux/auto.txt"), loadProbeInfo, *flagArches, &clangtool.Config{
```

Comment:
Nit: It might be better to parse the input (split the arches) here and pass the slice/map of arches as a parameter rather than do it at the bottom of the call stack.

Feel free to ignore this nit.

Reply:
Done. PTAL.